src/util/ec.rs

```diff
@@ -62,60 +62,34 @@ mod tests {
         let expected_pc = "09c02e309a8ac06d644aab0c8b52dd4318825340d7a33336a0b496a21cbea56229";
         assert_eq!(expected_pc, pc.to_string());
 
+        // 2. Create and verify range proofs - https://docs.rs/secp256k1-zkp/0.9.2/secp256k1_zkp/struct.RangeProof.html
```

Comment:
looks like we can also get param values for range proofs to be used in liquid.rs from - https://github.com/ElementsProject/elements/blob/2d298f7e3f76bc6c19d9550af4fd1ef48cf0b2a6/src/blind.cpp#L246-L265
ex: `min_bits = 52` would cover 21 million bitcoin (larger range requires a larger proof - so we could also adjust that i guess), `message` to convey asset info, if we're supporting unspendable scriptPubKey `min_value` would be 0, otherwise 1.

more docs - https://github.com/ElementsProject/elements/blob/master/doc/elements-confidential-transactions.md#range-proofs-1

Reply:
we wont be supporting unspendable scriptPubKey, so i guess `min_value` its always 1 for us.

Reply:
although i wonder if `unspendable` in this context refers to an output that is not ours...

Reply:
i assumed it was referring to `freezing` our own utxos and making them `unspendable`